code-challenges.js

```diff
@@ -31,38 +58,101 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+let  givenNum = (temperature1) => {
+  if(temperature1 <= 212) {
+    return `"${temperature1} is below boiling point"`
+  } 
+    else if(temperature2 >= 212) {
+    return `"${temperature2} is above boiling point"`
+  } 
+    else if(temperature3 === 212) {
+    return `"${temperature3} is at boiling point"`
+  } 
+    else {
+    return "something is wrong"
+  }
+}
+console.log(givenNum(temperature1))
+console.log(givenNum(temperature2))
+console.log(givenNum(temperature3))
+//I AM STUCK HERE-----I can't get temp3 to log at boiling point
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
 
+// input: combine two arrays
+// output: arrays are added together to equal length
+// create a parameter that takes two arrays of data types
+// combine length of arrays
+// return the combined length
+
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+const combinedRuns = [padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)]
```

Comment:
You wouldn't need this line of combinedRuns since you are using concat below and then console.log that combinedArrays variable.

Reply:
So it would be?:

// Pseudo code:
// input: combine two arrays
// output: arrays are added together to equal length
// Create a variable with a parameter that takes in two arrays of data types and combine length of arrays by using .concat method
// return one combined arrays length

  let combinedArrays = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)

console.log(combinedArrays.length)

Reply:
Exactly!